guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -9,7 +9,7 @@ All subscription information is available in your Red Hat Customer Portal accoun
 Before you can complete the tasks in this chapter, you must create a Red{nbsp}Hat Subscription Manifest in the Customer Portal.
 
 ifdef::satellite[]
-Note that the subscription model is deprecated and will be removed in a future release.
+Note that legacy subscription management, including the need to attach subscriptions to hosts and activation keys, is deprecated and will be removed in a future release.
```

Comment:
I added this change as well because subscriptions themselves are not deprecated. Manifests still contain subscriptions, and subscriptions are still required in order to meet contractual obligations.

Reply:
Thanks for PR!

Question about this line: Are there right now two subscription management models - new and legacy - or is there only one subscription management model which will be changed in the future release, thus making the current workflow unsupported?

Reply:
Right now both SCA and non-SCA subscription models are available. Non-SCA is deprecated but will still be available for several releases. SCA is now the default and will eventually be the only option.